### DISCUSSION TITLE
Is there any way to disable prefetch with dynamic import?

### DISCUSSION THREAD

**Discussion by idoshamun** [vercel/next.js#21973] [Help]
Hey!

I am using dynamic import to conditionally load relevant components but I don't want to prefetch them.
This might stack up to a massive amount of unused scripts.
At the very least I would like to prioritize it.
I would love your help on this one because I couldn't find anything in the docs.

**Comment by bericp1** [comment]
Seconded, anyway to disable this or control the behavior? E.g. mark certain dynamic imports as truly lazy? We have a library we load lazily in an onChange handler to format a phone number input on our registration page and that library is being preloaded _on our homepage_ (which links to the registration page above the fold). This is resulting in a lot of downloaded javascript that goes unused, wasting users' data.

Is the only option to disable preloading for the whole registration page that's linked to from the homepage?

**Comment by AndersDJohnson** [comment]
Would love this as well. Seems like the browser & network does more work than we want it to do prefetching all the dynamic chunks upfront.

**Comment by ChendyRewa** [comment]
i would also love the solution for this issue , any luck for the solution ?
